pkg/manifests/config.go

```diff
@@ -303,6 +303,35 @@ func (c *Config) applyDefaults() {
 	if c.ClusterMonitoringConfiguration.PrometheusK8sConfig.CollectionProfile == "" {
 		c.ClusterMonitoringConfiguration.PrometheusK8sConfig.CollectionProfile = FullCollectionProfile
 	}
+
+	// defaultExcludedDevices is the default for 2 arguments "--collector.netclass.ignored-devices" and "--collector.netdev.device-exclude".
+	// 6 kinds of virtual NICs are ignored by default:
```

Comment:
```suggestion
	// The following virtual NICs are ignored by default:
```
Better not to specify a specific number in case the total number ever changes.